usr/bin/update-i2pbrowser

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+## Copyright (C) 2012 - 2018 ENCRYPTED SUPPORT LP <adrelanos@riseup.net>
+## See the file COPYING for copying conditions.
+
+if [ ! "$tpo_downloader_debug" = "" ]; then
+   if [ "$tpo_downloader_debug" -ge "1" ]; then
+      set -x
+   fi
+fi
+
+set -o pipefail
+set -o errtrace
+
+SCRIPTNAME="$(basename "$BASH_SOURCE")"
+IDENTIFIER="i2pbrowser-downloader"
+
+ICON="/usr/share/icons/anon-icon-pack/tbupdate.ico"
+
+update-torbrowser --i2p "$@"
```

Comment:
Ideally, this would be the only non-comment line in this script.

Do you think you can find a solution how to reasonably set SCRIPTNAME, IDENTIFIER, ICON from parse_cmd?